crates/ruff/src/rules/flake8_comprehensions/rules/unnecessary_map.rs

```diff
@@ -240,7 +249,7 @@ struct LateBindingVisitor<'a> {
     /// The arguments to the current lambda.
     parameters: &'a Parameters,
     /// The arguments to any lambdas within the current lambda body.
-    lambdas: Vec<&'a Parameters>,
+    lambdas: Vec<Option<&'a Parameters>>,
```

Comment:
Would it make sense to only store existing parameters? Or is it important to know how many lambdas have a `None` parameter?

Reply:
We need to know if there are _any_ lambdas, i.e., whether this vector is non-empty.